lib/services/api/api_service.ts

```diff
@@ -172,6 +174,15 @@ export class SNApiService extends PureService {
     return response;
   }
 
+  async signOut() {
+    const url = await this.path(REQUEST_PATH_LOGOUT);
+    await this.httpService!.postAbsolute(
+      url,
+      {},
```

Comment:
This should be `undefined`. If postAbsolute doesn't accept undefined, it should be modified to do so.